### DISCUSSION TITLE
Can I test play on my Android?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [stride3d/stride#1342] [Q&A]
Hi, really cool game engine! :D Im testing out the JumpyYet demo project. Is there any way that I can conveniently start a debugging session on an android device connected via USB? 

Im considering to use Stride to make some mobile games for Android, the goal would be to be able to make code changes or changes to the level on my PC and quickly be able to see those changes on my Android device without having to wait too long. Would be especially important for games that require multi-touch (like a left and right controller for instance) where a mouse cant really replace true touch whilst play testing :3

The only alternative game engine that Im considering rn is Unity - but I really dislike there business model and the fact that its closed source. Ive also looked at O3DE (formerly Lumberyard/Cryengine) & Unreal, but both seem somewhat bloated and inefficient (Unreal Engine in particular takes nearly 2 minutes to display changes made on a USB connected android device & spits out games with insanely large packaged games making it obsolete). Which is what have kinda brought me here <3

**Comment by tebjan** [comment]
Since stride uses the official Microsoft tool chain, you can refer to this documentation for android: https://docs.microsoft.com/en-us/xamarin/android/deploy-test/debugging/debug-on-device?tabs=windows
